app/src/common/actions/article-detail.ts

```diff
@@ -59,13 +54,13 @@ function failSaveDetail(error: string): ArticleDetailAction {
 }
 
 /** Deletes an article */
-export function deleteArticle(id: number): any {
+export function deleteArticle(id: number, article: Article): any {
```

Comment:
It was to pass an "empty" article to prevent the red error screen after usign the delete function you did.